contexts/DonationContext/src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -60,7 +60,7 @@ public function addDonation( AddDonationRequest $donationRequest ): AddDonationR
 
 		$donation = $this->newDonationFromRequest( $donationRequest );
 
-		if ( $this->policyValidator->needsModeration( $donationRequest ) ) {
+		if ( !$donation->hasExternalPayment() && $this->policyValidator->needsModeration( $donationRequest ) ) {
```

Comment:
At this point I'd have factored out the predicate into a donationNeedsModeration method